config/kubernetes/sig-release/teams.yaml

```diff
@@ -94,31 +94,94 @@ teams:
     - xmudrii # 1.15 Bug Triage Shadow
     - zacharysarah # Docs
     privacy: closed
-  kubernetes-release-managers:
-    description: People actively working on next k8s minor release.  Gives admin access to repos
-      where branches must be created, etc., and write access to ones where label/PR
-      management is needed. Remove users who are not actively doing this job.
+  licensing:
+    description: Members of the Licensing subproject.
     members:
+    - dims # subproject owner
+    - justaugustus # subproject owner
+    - nikhita # subproject owner
+    - swinslow # subproject owner
+    privacy: closed
+  release-engineering:
+    description: Members of the Release Engineering subproject.
+    members:
+    - aleksandra-malinowska # subproject owner
+    - calebamiles # subproject owner
+    - hoegaarden # subproject owner
+    - ixdy # subproject owner
+    - tpepper # subproject owner
+    privacy: closed
+    teams:
+      kubernetes-release-managers:
```

Comment:
Nested teams have this behaviour. Just want to double check if you are ok with this, because I _think_ it can get weird with permissions.

> Child teams inherit the parent's access permissions, simplifying permissions management for large groups. Members of child teams also receive notifications when the parent team is @mentioned, simplifying communication with multiple groups of people.

Ref: https://help.github.com/en/articles/about-teams#nested-teams

Reply:
Yep, yep, I reviewed the same page. That's exactly what I want i.e., a top-level pingable group for releng, that we can assign permissions to, if need be, and have them cascade into the existing groups. :)